Src/LinearSolvers/MLMG/AMReX_MLABecLaplacian.cpp

```diff
@@ -147,6 +186,15 @@ MLABecLaplacian::setBCoeffs (int amrlev, Real beta)
     m_needs_update = true;
 }
 
+/**
+ * For each component in the supplied vector, this function
+ * sets beta as a scalar constant to the same value of the corresponding
+ * component across the multifab.
```

Comment:
I don't mean to be nit-picky, but I find this awkward. Maybe something like this is clearer?

```suggestion
 * Set each beta component to a scalar constant value corresponding to the 
 * respective component of the supplied vector.
```